icon_validator/rules/plugin_validators/help_example_validator.py

```diff
@@ -1,75 +1,75 @@
 import json
 import re
+from typing import List, Tuple
+
+from icon_plugin_spec.plugin_spec import KomandPluginSpec
+
 from icon_validator.exceptions import ValidationException
 from icon_validator.rules.validator import KomandPluginValidator
 
 
 class HelpExampleValidator(KomandPluginValidator):
-    validate_errors = []
-    pattern = r"#### (.*?)\n\n.*?Example input:\n\n```\n(.*?)\n\n#.*?Example output:\n\n```\n(.*?)\n\n#"
+    pattern = r"\n[#]{4} (.*?)\n.*?Example input:\n\n(```.*?```)\n.*?Example output:\n\n(```.*?```)\n"
 
-    @staticmethod
-    def get_objects(function_pattern, raw_help):
-        raw_object = re.findall(function_pattern, raw_help, re.DOTALL)
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.validate_errors: List[ValidationException] = []
 
+    def get_objects(self, function_pattern: str, raw_help: str) -> List[Tuple[str]]:
+        raw_object = re.findall(function_pattern, raw_help, re.DOTALL)
         if not raw_object:
             return
+        return re.findall(self.pattern, raw_object[0], re.DOTALL)
 
-        return re.findall(HelpExampleValidator.pattern, raw_object[0], re.DOTALL)
-
-    @staticmethod
-    def add_json_error(input_output, function_type, title):
-        HelpExampleValidator.validate_errors.append(
+    def add_json_error(self, input_output: str, function_type: str, title: str) -> None:
+        self.validate_errors.append(
             f"Invalid JSON example identified in Example {input_output} of {function_type} titled {title}. "
```

Comment:
Does {title} correctly output the title of the action/trigger/task in the tests? In some instances I know it seems to select input or output as the title.

Reply:
yes, now it's working properly, I've fixed it.